apps/src/StudioApp.js

```diff
@@ -1784,11 +1808,16 @@ StudioApp.prototype.configureDom = function (config) {
     visualizationColumn.className = visualizationColumn.className + " centered_embed";
   }
 
+  var smallFooter = document.querySelector('#page-small-footer .small-footer-base');
+  if (config.noPadding && smallFooter) {
+    // The small footer's padding should not increase its size when not part
+    // of a larger page.
+    smallFooter.style.boxSizing = "border-box";
+  }
```

Comment:
Are we able to set this styling within in React component itself?

i.e. add it to the styles.smallFooter used here

` <div className={this.props.className} style={styles.smallFooter}>`